schematic/utils/df_utils.py

```diff
@@ -4,17 +4,58 @@
 
 import logging
 from copy import deepcopy
-from time import perf_counter
-from typing import Union, Any, Optional
 from datetime import datetime
+from time import perf_counter
+from typing import Any, Optional, Union
+
 import dateparser as dp
-import pandas as pd
 import numpy as np
+import pandas as pd
 from pandarallel import pandarallel  # type: ignore
 
+# pylint:disable=no-name-in-module
+from pandas._libs.parsers import STR_NA_VALUES  # type: ignore
+
+STR_NA_VALUES_FILTERED = deepcopy(STR_NA_VALUES)
+
+try:
+    STR_NA_VALUES_FILTERED.remove("None")
+except KeyError:
+    pass
+
 logger = logging.getLogger(__name__)
 
 
+def read_csv(
```

Comment:
should we give this function a slightly different name to avoid confusion? i.e. `read_csv_no_none` or `read_csv_filter_none`?

Reply:
I'm not sure if we should change the name of the function. Depending on the `keep_default_na` bool is what determines if `None` is going to be filtered out. It would be a bit awkward to call a function called `read_csv_no_none` if you aren't using that functionality.